benchmarks/DeepLearning/Models/MobileNet-V3/README.md

```diff
@@ -0,0 +1,49 @@
+# Buddy Compiler MobileNetV3 Benchmark
```

Comment:
Let’s not make the READMEs too scattered. Remove this file and add the necessary information (python env and path) to the deep learning benchmark README.